server/api/routes/auth.js

```diff
@@ -43,6 +46,22 @@ router.get('/callback', (req, res, next) => {
       if (error) {
         return next(error);
       }
+      
+      // store user_id?
+      const userInfo = {
+        name: user.displayName,
+        identifier: user.id
+      };
+      
+      // User.createUser(userInfo);
+      User.findUser(user.id).then(data => {
+        if (data.length === 0) {
+          User.createUser(userInfo);
```

Comment:
```suggestion
          users.createUser(userInfo);
```

same question here about using `await`